Localization/es-ES/Mods.TerrariaOverhaul.Configuration.hjson

```diff
@@ -159,18 +159,18 @@ Balance: {
 	}
 
 	EnableDamageRandomizationRemoval: {
-		// DisplayName: "Enable Damage Randomization Removal"
-		// Description: "When enabled, all damage numbers in the game will be made fair and predictable."
+		DisplayName: "Activar remoción de randomización de daño"
```

Comment:
I would prefer "eliminación de aleatorización" since "randomización" doesn't exist in Spanish and "remoción" is not a synonym of eliminate

Reply:
Hmmm, how about "eliminación de variación en daño"? I'm thinking "aleatorización" sounds a little wordy

Reply:
Sorry, didn't check the comment. I'd go for it too, sounds more organic, yep